platform/web-security/src/main/resources/META-INF/motech/securityContext.xml

```diff
@@ -151,4 +151,10 @@
         <constructor-arg value="/module/server/changepassword"/>
     </bean>
 
+    <bean id="viewResolver" class="org.springframework.web.servlet.view.InternalResourceViewResolver">
```

Comment:
I wanted to move it away from osgi directory, since it's not OSGi related. I should have moved it to application context file though, my bad. Will fix.